bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/BookieIsReadyService.java

```diff
@@ -51,7 +51,7 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
         StateManager sm = bookie.getStateManager();
         if (sm.isRunning() && !sm.isShuttingDown()) {
             response.setCode(HttpServer.StatusCode.OK);
-            response.setBody("");
+            response.setBody("Bookie is fully started.");
```

Comment:
What about a simple 'OK'?

Not a big deal.

BTW, can you update/add the tests?